app/javascript/dashboard/routes/dashboard/commands/commandBarActions.js

```diff
@@ -30,68 +31,37 @@ export const OPEN_CONVERSATION_ACTIONS = [
   },
 ];
 
+const createSnoozeHandlers = busEventName => {
+  return Object.values(SNOOZE_OPTIONS).map(option => ({
+    id: option,
+    title: `COMMAND_BAR.COMMANDS.${option.toUpperCase()}`,
+    parent: 'snooze_conversation',
+    section: 'COMMAND_BAR.SECTIONS.SNOOZE_CONVERSATION',
+    icon: ICON_SNOOZE_CONVERSATION,
+    handler: () => bus.$emit(busEventName, option),
+  }));
+};
+
+// Actions for the standard conversation
 export const SNOOZE_CONVERSATION_ACTIONS = [
   {
     id: 'snooze_conversation',
     title: 'COMMAND_BAR.COMMANDS.SNOOZE_CONVERSATION',
     icon: ICON_SNOOZE_CONVERSATION,
     children: Object.values(SNOOZE_OPTIONS),
   },
+  ...createSnoozeHandlers(CMD_SNOOZE_CONVERSATION),
+];
 
+// Actions for the chat list snooze options like bulk actions and the context menu
+export const CHAT_LIST_SNOOZE_CONVERSATION_ACTIONS = [
```

Comment:
I created separate snooze conversation events for the bulk action/context menu to address a few challenges I encountered:

**Challenges Faced**

1. Initially, I considered creating a separate command hotkeys mixin for bulk actions but decided against it. It seemed excessive since we could directly integrate it into the conversationHotKeys mixin with some minor changes.
2. I attempted to reuse the same event from the resolve action dropdown for snoozing conversations. However, I dropped this approach because it led to an issue where the same bus events were triggered twice (this only occurred when the chat was open) when snoozing from the bulk action menu. This duplication happened because the bus event is tied to the `ResolveAction.vue` component, which is always visible.
3. To resolve the issue mentioned above, I tried creating a separate component for the resolve action dropdown and relocated the snooze bus event to it. This change would ensure that the bus event was only active when the resolve action dropdown was visible. Nonetheless, I had to drop this solution because it failed when using the command bar by opening from the `cmd+k` key shortcut to search for and execute a snooze on a conversation because the bus event would not be active unless the resolve action dropdown was visible.
cc @pranavrajs @muhsin-k @scmmishra